app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/IMapView.java

```diff
@@ -21,5 +22,9 @@
     rx.Observable<ViewerCamera> getViewerCameraObservable();
 
     void updateMapEntity(EntityUpdateEventArgs entityUpdateEventArgs);
+
+    void showVectorLayer(VectorLayerPresentation vectorLayerPresentation);
+
+    void hideVectorLayer(String vectorLayerId);
 }
 
```

Comment:
It's a bit weird that IMapView has almost the same virtual methods as GGMap isn't it? I must admit that i'm not familiar with these areas in the code but it looks like they (the 2 interfaces) are pretty much the same, and indeed the ViewerFragment only delegates those mutual methods to its own GGMap field. Why is that?